api/v1alpha1/reaper_types.go

```diff
@@ -37,6 +37,15 @@ const (
 	DefaultStorageType = StorageTypeMemory
 )
 
+type InitContainerConfig struct {
+
+	// SecurityContext applied to the Reaper init container
+	SecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
+
+	// VolumeMounts available to the Reaper init container
+	VolumeMounts []corev1.VolumeMount `json:"volumeMounts,omitempty"`
```

Comment:
We don't need to expose volume mounts here. What we do need is to create a volume mount for `/etc/reaper` or whatever the config location is.

Reply:
👍🏻